src/core/dbus-execute.c

```diff
@@ -1792,6 +1840,55 @@ int bus_exec_context_set_transient_property(
         if (streq(name, "LogRateLimitBurst"))
                 return bus_set_transient_unsigned(u, name, &c->log_ratelimit_burst, message, flags, error);
 
+        if (streq(name, "LogFilterPatterns")) {
+                _cleanup_strv_free_ char **allow_list = NULL, **deny_list = NULL;
+                _cleanup_(pattern_freep) pcre2_code *compiled_pattern = NULL;
```

Comment:
`compiled_pattern` must be declared in the `while` loop below, otherwise, memleak.